news/DEADBEEF.trivial

```diff
@@ -0,0 +1 @@
+
```

Comment:
Nice name. 😂

Reply:
Yea, my embedded GUID generator crashes to generate anything longer. :D